.github/actions/find-runner.js

```diff
@@ -0,0 +1,18 @@
+const { Octokit } = require("@octokit/action");
+
+const octokit = new Octokit();
+
+get_async()
+
+async function get_async () {
```

Comment:
Maybe define the function before it's used? Also, not clear what it's getting async so a better name would be nice.